LaunchDarkly/LaunchDarkly/Models/LDConfig.swift

```diff
@@ -31,12 +35,21 @@ public enum LDStreamingMode {
  you can use targeting rules to enable "dark mode" for all customers who are using version 15 or greater, and ensure
  that customers on previous versions don't use the earlier, unfinished version of the feature.
  */
+#if os(Linux) || os(Windows)
+public enum AutoEnvAttributes: Int {
+     /// Enables the Auto EnvironmentAttributes functionality.
+    case enabled
+    /// Disables the Auto EnvironmentAttributes functionality.
+    case disabled
+}
+#else
 @objc public enum AutoEnvAttributes: Int {
     /// Enables the Auto EnvironmentAttributes functionality.
     case enabled
     /// Disables the Auto EnvironmentAttributes functionality.
     case disabled
 }
+#endif
```

Comment:
Couldn't you put `@objc` on it's own line and just wrap that one line in the `#if`?

Reply:
Oh hmm, let me try that!

Reply:
Looks like this doesn't work since the swift compiler fails to parse it as one expression sadly 
<img width="914" alt="image" src="https://github.com/thebrowsercompany/ios-client-sdk/assets/63919/cf15cc46-c513-4f27-9c92-81acbf8b5b62">

Reply:
```
#if !os(linux) && !os(Windows)
@objc
#endif
public enum AutoEnvAttributes: Int {
...
```?